.github/workflows/manual_msi_build.yml

```
@@ -71,18 +71,18 @@ jobs:
     runs-on: windows-2022
     steps:
       - name: Checkout Sources
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Setup Go
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
         with:
           go-version: "1.20"
           check-latest: true
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v4
+        uses: goreleaser/goreleaser-action@v5
         with:
           # either 'goreleaser' (default) or 'goreleaser-pro'
           distribution: goreleaser-pro
-          version: "v1.18.2"
+          version: "v1.22.1"
           args: build --single-target --skip-validate --clean --snapshot
```

I might have missed more of these 😄 
```suggestion
          version: "v1.22.1"
          args: build --single-target --skip=validate --clean --snapshot
```

---

Good catch, I'll double check them all.